app/styles/_base.scss

```diff
@@ -98,8 +98,28 @@ a {
       top: auto;
     }
   }
+
+  :global &.-underline {
```

Comment:
@andresgnlez there's no way here to NOT have the same 20 lines duplicated just to have the "global"?